src/ResultUtils/OptionCE.fs

```diff
@@ -0,0 +1,73 @@
+namespace ResultUtils
+
+open System
+
+[<AutoOpen>]
+module OptionCE =
+  type OptionBuilder() =
```

Comment:
indentation is off here, should be 4 spaces, not 2

Reply:
Fixed.